doc/classes/Resource.xml

```diff
@@ -76,6 +76,7 @@
 		<signal name="changed">
 			<description>
 				Emitted whenever the resource changes.
+				[b]Note:[/b] This signal is not emitted automatically on change, which means that for custom resources you need to create a setter and emit the signal yourself.
```

Comment:
IIUC it *is* emitted automatically on change (otherwise it's useless), but only for builtin resources. It's the resource's responsibility to emit it whenever it changes.

This makes it sound like it's never emitted.

Reply:
Yeah it is emitted, but explicitly with `emit_changed()` method. So built-in resources just handle it internally.

Reply:
Yes but this is documentation for signals that can be connected to in Resources. Most of the time you use builtin resources, and this gives you documentation on what signal from those builtin Resources you can handle. As it's phrased, it sounds like you won't get anything by connecting to this signal, which is not true for builtin Resources.

Reply:
Ok changed.